<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>open_image</class>
 <widget class="QMainWindow" name="open_image">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="browse_image">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>browse_image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="apply_harris">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>apply_harris</string>
    </property>
   </widget>
   <widget class="QPushButton" name="apply_sift">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>apply_sift</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>791</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGraphicsView" name="original_graphicsView"/>
     </item>
     <item>
      <widget class="QGraphicsView" name="harris_graphicsView"/>
     </item>
     <item>
      <widget class="QGraphicsView" name="sift_graphicsView"/>
     </item>
    </layout>
   </widget>
   <widget class="QSlider" name="horizontalSlider_harris_threshold">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="harris_threshold">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Harris Threshold:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="upload_2nd_img">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>upload_2nd_img</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_2nd_img">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>261</width>
      <height>199</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_matching">
    <property name="geometry">
     <rect>
      <x>288</x>
      <y>310</y>
      <width>521</width>
      <height>199</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="apply_match">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>280</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>apply_match</string>
    </property>
   </widget>
   <widget class="QLabel" name="match_result">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>510</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>result</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>843</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
